resolwe_bio/processes/reads_processing/cutadapt.yml

```diff
@@ -205,130 +216,131 @@
-      description: >
-        Interpret IUPAC wildcards in reads.
-    - name: times
-      label: Times
-      type: basic:integer
-      required: false
-      default: 1
-      description: >
-        Remove up to COUNT adapters from each read.
-    - name: error_rate
-      label: Error rate
-      type: basic:decimal
-      required: false
-      default: 0.1
-      description: >
-        Maximum allowed error rate (no. of errors divided by the length of the matching region).
+    - name: adapters
+      label: Adapters
+      group:
+        - name: up_primers_file_fwd
+          label: 5 prime forward reads adapter file
+          type: data:seq:nucleotide
+          required: false
+        - name: down_primers_file_fwd
+          label: 3 prime forward reads adapter file
+          type: data:seq:nucleotide
+          required: false
+        - name: up_primers_file_rev
+          label: 5 prime reverse reads adapter file
+          type: data:seq:nucleotide
+          required: false
+        - name: down_primers_file_rev
+          label: 3 prime reverse reads adapter file
+          type: data:seq:nucleotide
+          required: false
+        - name: up_primers_seq_fwd
+          label: 5 prime forward reads adapter sequence
+          type: list:basic:string
+          required: false
+        - name: down_primers_seq_fwd
+          label: 3 prime forward reads adapter sequence
+          type: list:basic:string
+          required: false
+        - name: up_primers_seq_rev
+          label: 5 prime reverse reads adapter sequence
+          type: list:basic:string
+          required: false
+        - name: down_primers_seq_rev
+          label: 3 prime reverse reads adapter sequence
+          type: list:basic:string
+          required: false
+        - name: times
+          label: Times
+          type: basic:integer
+          required: false
+          default: 1
+          description: >
+            Remove up to COUNT adapters from each read.
+        - name: error_rate
+          label: Error rate
+          type: basic:decimal
+          default: 0.1
+          description: >
+            Maximum allowed error rate (no. of errors divided by the length of the matching region).
+        - name: min_overlap
+          label: Minimal overlap
+          type: basic:integer
+          default: 3
+          description: >
+            Minimum overlap for an adapter match
+        - name: match_read_wildcards
+          label: Match read wildcards
+          type: basic:boolean
+          required: false
+          default: false
+          description: >
+            Interpret IUPAC wildcards in reads.
+    - name: modify_reads
+      label: Modifying reads
+      group:
+        - name: leading
+          label: Leading quality
+          type: basic:integer
+          required: false
+          description: >
+            Remove low quality bases from the beginning. Specifies the minimum quality required to keep a base.
```

Comment:
Consider joining the leading and trailg parameters in one, as it is originally done in CLI interface. Explain the difference between notation with and without a comma. Also, use "3/5 prime" notation, leading/trailing can be confusing.

Reply:
I think it as it is might be more clear to biologist user and not too confusing for bioinformaticians. However I agree about improving ``label`` notation. I will leave it as it is and correct it when we figure out user stories.